.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+images/android-base @mapbox/maps-android
```

Comment:
seems I don't have admin to this project, and cannot add write permission to this group. cc @tobrun